### DISCUSSION TITLE
Framework properties for Question are hardcoded in the code for BMGS. It should be configurable.

### DISCUSSION THREAD

**Discussion by vaivk369** [Sunbird-Knowlg/Community#105] [General Discussions]
https://github.com/orgs/Sunbird-Ed/discussions/420 has stated the issue. The Framework properties for the question is hard-coded and if we have to add new or edit existing we need to edit the code. It should be configurable on category definition level or form level like we have for questionset

@pallakartheekreddy

**Comment by pallakartheekreddy** [comment]
@vaivk369 This issue needs to be resolved in the question set editor, not in the collection editor, as the question set editor code is separated from the collection editor. Please create this thread in inQuiry BB. 
@Krishnaj20

**Reply by pallakartheekreddy** [reply]
@vaivk369 Pls let us know if your query is resolved

**Reply by pallakartheekreddy** [reply]
@vaivk369 As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
